scripts/configs/drone/main.jsonnet

```diff
@@ -87,25 +87,38 @@ local docker_step(tag, os, arch, version='') =
     settings: {
       repo: docker_repo,
       dry_run: 'true',
+      target: if with_browser then 'with-browser' else 'release',
       build_args: [
         'TARGETPLATFORM=' + platform,
         'TARGETOS=' + os,
         'TARGETARCH=' + arch,
       ] + if std.length(version) > 0 then [
         'TARGETVARIANT=' + version,
+      ] else []
+       + if with_browser then [
+        'WITH_BROWSER=true',
```

Comment:
What does this build arg do? I'm not finding it in the dockerfile 🤔

Reply:
Thank you for catching this -- I started with an ARG-based solution first and then migrated to a built target one, so this needs to be removed